yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -132,6 +133,16 @@ func InitAssessmentDB() error {
 			parent_table_name   TEXT,
 			size_in_bytes       INTEGER,
 			PRIMARY KEY(schema_name, object_name));`, TABLE_INDEX_STATS),
+		fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
+			queryid			BIGINT,
+			query		TEXT,
+			calls		BIGINT,
+			total_exec_time REAL,
```

Comment:
we are not using any of these columns other than `query` as of now, right?

Reply:
yeah, i just kept those to be future proof.
Other columns we might use for Performance Validations